src/App.js

```diff
@@ -66,7 +66,7 @@ class App extends Component {
               exact
               path='/dashboard' 
               render={props => (
-                <Dashboard {...props} loggedIn={this.state.loggedIn} loggedUsername={this.state.username} loggedId={this.state.id}/>
+                <Dashboard {...props} user={this.state} loggedIn={this.state.loggedIn} loggedUsername={this.state.username} loggedId={this.state.id}/>
```

Comment:
do we need to retain loggedUsername/loggedIn as a passed prop if we are passing a user object already?

Reply:
Oops I meant to remove all of those when I updated to passing a user object, good catch!